database scripts - mysql/Itembank/Stored Procedures/updatetdsconfigs.sql

```diff
@@ -60,7 +60,13 @@ begin
 		select tpp.propvalue 
 		from loader_testpackageproperties tpp 
 		where tpp.propname = 'msb'), '0') = '1' then   
-			update configs.client_testproperties ctp set msb = 1 where ctp.testid = v_testname;
+			update configs.client_testproperties ctp set msb = 1 
```

Comment:
I think it should have been this restrictive in the first place to line it up with the rest of the system.